tools/test/psa/__init__.py

```diff
@@ -1,17 +0,0 @@
-"""
```

Comment:
Commit "Remvoe PSA related scripts from tools/test folder" should say "Remove PSA related scripts from tools/test folder"

Reply:
I'll squash the commits and update the comment.